### ISSUE TITLE
Drop autotools files

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/dballe#281]
Remove autotools files after the first successful release of a RPM package with Meson (see #269).

**Comment by edigiacomo** [comment]
I've started to fix this issue in branch `issue281`. @spanezz can I safely remove the following files?

- `libdballef.m4`
- `libdballe.m4`
- `libdballef.pc.in`
- `libdballe.pc.in`

**Comment by spanezz** [comment]
Yes, you can safely remove them: the `.m4` files were unused and should be gone, while the `.pc` files are generated by meson
